### ISSUE TITLE
RFE: keybindings

### ISSUE THREAD

**Issue by alhirzel** [KristofferC/OhMyREPL.jl#168]
I think it would be cool to be able to add a keybinding for e.g. running `fieldnames(typeof(<current_repl_text>))` or a similar smarter alias when a shortcut key like CTRL+L is pressed. How realistic is this to add into OhMyREPL.jl?

**Comment by KristofferC** [comment]
After #395, OhMyREPL is a thin layer over the native REPL; custom keybindings can be added via `REPL.LineEdit` keymaps in startup.jl, so this is out of scope here.
